pins.go

```diff
@@ -0,0 +1,9 @@
+package drivers
+
+// PinOutput is hardware abstraction for a pin which outputs a
+// digital signal (high or low voltage).
+type PinOutput func(level bool)
```

Comment:
```suggestion
type PinOutput func(level bool) error
```
I know, this partially violates the intention of this function - keep the same syntax like "machine-pin.Set()" to keep the changes as simple as possible, but when using the [character device driver](https://github.com/warthog618/go-gpiocdev) on Linux-machines, e.g. in gobot, setting an output can fail - same for the "Get()" function. Maybe there is another possibility to catch this problem?

Reply:
Today all TinyGo pin HAL does not return an error. We could think of adding a PinOutputFailer or similar in the future, I'd avoid too much scope in one PR for now.